backend/driver.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright © 2009-2013 Bernhard M. Wiedemann
+#~ Copyright © 2009-2013 Bernhard M. Wiedemann
```

Comment:
This change looks unintended but you could update the year in the line below.

Reply:
done